src/updater/updater.cpp

```diff
@@ -120,10 +120,16 @@ auto mmapFile(const std::string &filename, boost::interprocess::mode_t mode)
     using boost::interprocess::file_mapping;
     using boost::interprocess::mapped_region;
 
+    {
+        storage::io::FileReader file(filename, storage::io::FileReader::VerifyFingerprint);
```

Comment:
Okay just looked at the [implementation](https://github.com/Project-OSRM/osrm-backend/blob/master/include/storage/io.hpp#L28): We just wrap `ifstream` which closes on destruction, which still begs the question of why this files here.